west.yml

```diff
@@ -52,7 +52,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 79b77b4f6130d0db539a6f6655b3e03818ab98a6
+      revision: pull/789/head
```

Comment:
Please update SHA: `4ab692bec88a143f89f2abfdc086e3f8087622f4`

Reply:
Ok Rebased and Updated.